.tekton/push.yaml

```diff
@@ -1,43 +0,0 @@
----
-apiVersion: tekton.dev/v1beta1
-kind: PipelineRun
-metadata:
-  name: integration-service-on-push
-  annotations:
-    pipelinesascode.tekton.dev/on-event: "[push]" 
-    pipelinesascode.tekton.dev/on-target-branch: "[main]"
-    pipelinesascode.tekton.dev/max-keep-runs: "5"
-spec:
-  params:
-    - name: git-url
-      value: "{{ repo_url }}"
-    - name: revision
-      value: "{{ revision }}"
-    - name: output-image
-      value: 'quay.io/redhat-appstudio/integration-service:{{ revision }}'
-    - name: infra-deployment-update-script
```

Comment:
Where is this updating mechanism now?

Reply:
I think it is done [here](https://docs.google.com/document/d/1nCHMYxyJQW_fAH8z97uk16IvYTQaodbUn87fMGgw8YA/edit#heading=h.ivxgstl1y3e7)

Reply:
Cool, thanks